services/121-service/test/helpers/registration.helper.ts

```diff
@@ -13,6 +15,56 @@ import {
   getServer,
 } from '@121-service/test/helpers/utility.helper';
 
+export function createOcwRegistrationForImport({
+  referenceId,
+  paymentAmountMultiplier,
+  addressHouseNumber,
+  fullName,
+  phoneNumber,
+  whatsappPhoneNumber,
+  addressStreet,
+  addressPostalCode,
+  addressCity,
+  programFinancialServiceProviderConfigurationName,
+}: {
+  referenceId?: string;
+  paymentAmountMultiplier?: number;
+  addressHouseNumber?: number | null;
+  fullName?: string | null;
+  phoneNumber?: string | null;
+  whatsappPhoneNumber?: string | null;
+  addressStreet?: string | null;
+  addressPostalCode?: string | null;
+  addressCity?: string | null;
+  programFinancialServiceProviderConfigurationName?: string;
+}) {
+  return {
+    referenceId:
+      referenceId !== undefined
+        ? referenceId
+        : `ref-${Math.random().toString(36).substring(2, 10)}`,
+    preferredLanguage: LanguageEnum.en,
+    paymentAmountMultiplier:
+      paymentAmountMultiplier !== undefined ? paymentAmountMultiplier : 1,
+    fullName: fullName !== undefined ? fullName : 'Default Name',
+    phoneNumber: phoneNumber !== undefined ? phoneNumber : '14155236666',
+    programFinancialServiceProviderConfigurationName:
+      programFinancialServiceProviderConfigurationName
+        ? programFinancialServiceProviderConfigurationName
+        : FinancialServiceProviders.intersolveVisa,
+    whatsappPhoneNumber:
+      whatsappPhoneNumber !== undefined ? whatsappPhoneNumber : '14155236666',
+    addressStreet:
+      addressStreet !== undefined ? addressStreet : 'Default Street',
+    addressHouseNumber:
+      addressHouseNumber !== undefined ? addressHouseNumber : 1,
+    addressHouseNumberAddition: '',
+    addressPostalCode:
+      addressPostalCode !== undefined ? addressPostalCode : '1234AB',
+    addressCity: addressCity !== undefined ? addressCity : 'Default City',
```

Comment:
```suggestion
    referenceId: referenceId ?? `ref-${Math.random().toString(36).substring(2, 10)}`,
    preferredLanguage: LanguageEnum.en,
    paymentAmountMultiplier: paymentAmountMultiplier ?? 1,
    fullName: fullName ?? 'Default Name',
    phoneNumber: phoneNumber ?? '14155236666',
    programFinancialServiceProviderConfigurationName: programFinancialServiceProviderConfigurationName ?? FinancialServiceProviders.intersolveVisa,
    whatsappPhoneNumber: whatsappPhoneNumber ?? '14155236666',
    addressStreet: addressStreet ?? 'Default Street',
    addressHouseNumber: addressHouseNumber ?? 1,
    addressHouseNumberAddition: '',
    addressPostalCode: addressPostalCode ?? '1234AB',
    addressCity: addressCity ?? 'Default City',
```

Curious; would the above have the same effect?

Reply:
Almost, however I want to also explicitly be able to set variables to null. So that the default value is not used. For example import addressHouseNumber as null instead of 1.

Reply:
This file is also a bit out of scope for this PR. So resolving it for now

Reply:
Yeah, the "explicitly `null`" scenario cannot be done with this short `??` syntax no.. too bad..